packages/compiler/src/source.ts

```diff
@@ -1,31 +1,92 @@
 import type { BOSModule } from '@bos-web-engine/common';
```

Comment:
I'm not following how these changes are related to the IndexedDB functionality. Can you please elaborate on why this would need to change now that compiler responses are being cached?

Reply:
yeah, sorry for making this a bit more complicated than it could have been.
The `BOSModule` interface was reused in [type ComponentSourcesResponse](https://github.com/near/bos-web-engine/pull/387/files#diff-96c03b19b7d8da1f7488fb14fa014d55b32ffe5d2fe9d9baab57bde3218470aaR148), which is used to cast the result of `prepareSourceWithBlockHeight` function. 
The idea was to keep the same output as it was before.